dwl.c

```diff
@@ -1222,6 +1224,21 @@ rendermon(struct wl_listener *listener, void *data)
 	struct timespec now;
 	clock_gettime(CLOCK_MONOTONIC, &now);
 
+	wl_list_for_each(c, &stack, slink) {
+		if (c->xdg_surface->role != WLR_XDG_SURFACE_ROLE_TOPLEVEL)
+			continue;
+		if (c->xdg_surface->toplevel->server_pending.width !=
+				c->xdg_surface->surface->current.width ||
+				c->xdg_surface->toplevel->server_pending.height !=
+				c->xdg_surface->surface->current.height) {
```

Comment:
Nice idea; shame we don't have position info in server_pending.

Reply:
Nah, position info isn't needed here, since Wayland clients are position-agnostic.  Resizing nicely, however, requires a round-trip to the client.

Reply:
Good to know; thanks!